wikimedia/wikipedia-ios
# Notifications translations tweaks

**Phabricator:** 
No specific Phabricator links, but these are based off of this information:
https://github.com/wikimedia/wikipedia-ios/pull/4170#issuecomment-1085096622
https://phabricator.wikimedia.org/T297620#7821627

### Notes
These are more tweaks that were requested over the last few weeks for improved translations.

7a887c2 - using the terms "discussion" and "collaboration" instead of "talk" for certain projects seemed to be an English-only adjustment, so we're removing that requirement (confirmed with @carolynlimadeo).
c173148 - Removing the use of project names within sentences. This requirement was confirmed in [this Phabricator comment](https://phabricator.wikimedia.org/T297620#7821627).
594eb80 - I noticed some notifications translations were not being extracted in our build script, so this fixes that.

### Test Steps
1. Ensure unit tests pass.
2. Super light regression on notifications cells labels, make sure nothing looks obviously wrong.

### Screenshots/Videos
![IMG_0709](https://user-images.githubusercontent.com/3620196/165313620-b9d76c88-d43b-4310-ba96-01577b35d7ee.PNG)

![IMG_0710](https://user-images.githubusercontent.com/3620196/165313647-58761c77-0231-4948-be4d-f69ecef4b28e.PNG)
